spec/integration/compliance/audit_spec.rb

```diff
@@ -0,0 +1,81 @@
+require "spec_helper"
```

Comment:
Can we get the specs renamed to compliance as well so it all matches?

Reply:
I thought for sure I had already done that, nice catch.